fiobank.py

```diff
@@ -104,7 +115,7 @@ def _parse_info(self, data):
     def _parse_transactions(self, data):
         schema = self.transaction_schema
         try:
-            entries = data['accountStatement']['transactionList']['transaction']
+            entries = data['accountStatement']['transactionList']['transaction']  # noqa
```

Comment:
It's exactly 80, which is the problem since pep states that you should "Limit all lines to a maximum of 79 characters." https://www.python.org/dev/peps/pep-0008/#maximum-line-length

The code should be at most 79 chars meaning that it will always fit to 80 chars screen...